backend/src/routes/db_lock/index.js

```diff
@@ -4,19 +4,19 @@ const what = "db_lock";
 const routes = [
   {
     method: "POST",
-    url: `/${what}`,
+    url: `/${what}/add`,
     // schema: validators.addOne,
     handler: controller.addLockByParams,
   },
   {
-    method: "GET",
-    url: `/${what}`,
+    method: "POST",
+    url: `/${what}/get`,
```

Comment:
This could be done with query params, so like `GET /db_lock?lockedBy={lockedBy}&lockedRowIds=...&lockedTable=...`, then we wouldn't have to use a POST request for what is more like a GET.